.github/workflows/performance.yml

```diff
@@ -0,0 +1,36 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
+
+jobs:
+  performance:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == '10up/wordpress-develop' || github.event_name == 'push' }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Docker debug information
+        run: |
+          docker -v
+          docker-compose -v
+
+      - name: Start Docker environment
+        run: npm run env:start
+
+      - name: Install WordPress
+        run: npm run env:install
```

Comment:
Let's go ahead and add steps here to run the new test:performance script along with a cleanup task to ensure no git tracked files are modified during the process ([example](https://github.com/10up/wordpress-develop/blob/trunk/.github/workflows/end-to-end-tests.yml#L107-L108))

Reply:
Added `npm run test:performance` and cleanup task.